e2e/testcases/apiservice_test.go

```diff
@@ -31,7 +32,8 @@ const (
 )
 
 func TestCreateAPIServiceAndEndpointInTheSameCommit(t *testing.T) {
-	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured, ntopts.RequireGKE(t))
+	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured,
```

Comment:
add comments as to why we're lengthening it, please.

Reply:
DONE
Also added a link to the prow job in the commit.